apps/financial-aid/web-osk/src/routes/application/RelationshipStatusForm/relationshipStatusForm.tsx

```diff
@@ -79,46 +94,71 @@ const RelationshipStatusForm = () => {
           Hjúskaparstaða þín
         </Text>
 
-        <Text marginBottom={[3, 3, 4]}>
-          Samkvæmt upplýsingum frá Þjóðskrá ert þú ekki í staðfestri sambúð. En
-          sért þú í óstaðfestri sambúð þarft þú og maki þinn bæði að skila inn
-          umsókn um fjárhagsaðstoð.
+        <Text variant="intro" marginBottom={[2, 2, 3]}>
```

Comment:
Did you think about having this as two pages? `InRelationshipForm` and `UnknownRelationshipForm`